javademo/README.md

```diff
@@ -0,0 +1,11 @@
+# Java demo
+
+## Prerequisites
+1. Running the demo requires a valid configuration. It can be set using environment variables in your system or IDE, or by sourcing an `.env` file. Refer to the example config found in [test_variables.env.example](../test_variables.env.example). Out of the variables listed there, the following are required:
+    - `DEMO_RPC_URL` - RPC node URL
+    - `DEMO_ACCOUNT_ADDRESS` - account address
+    - `DEMO_ACCOUNT_PRIVATE_KEY` - account private key
+2. Additionaly, you will need to compile a Cairo 0 balance contract and calculate its class hash.
+The path to compiled contract and class hash should be set manually in [Main.java](src/main/java/com/example/javademo/Main.java)
+
+To use all demo functionalities, your Starknet account should have some funds on it.
```

Comment:
Some info about the fact that TESTNET is recommended for testing (as the name suggests) and prefunding with [testnet faucet](https://faucet.goerli.starknet.io/) would be nice here, WDYT?

Reply:
I'm not sure it should be enforced, why not use integration? 

Also, should it be the responsibility of SDK's README to provide information about the faucet? This info would have to be added to 3 readmes (javademo, androidemo, main readme after #334). And technically, one can just conduct a web search for "how to obtain funds on the StarkNet testnet" 😄  

I don't mind adding it tho, if this is the approach used in starknet.py.

Reply:
We can add that info.
>but I'd rather just mention that the demo can be used with any rpc starknet node (including devnet)

However, demos can't be directly used with devnet because devnet has a different erc20 eth contract address. We could add instructions on modifying it locally or once again set it using environment variable, but I think that might create some extra confusion.

Reply:
Let's hardcode devnet addresses in the code then, and instruct users what to change in order to have a working demo with other nodes (that includes creating account on a network and prefunding it - without any specifics like "now for testnet erc20 address is 0x123 and for mainnet 0x321" - just general things you have to do/change). makes sense?

Reply:
Turns out the erc20 address on devnet is the same 👍🏻